src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -496,8 +496,15 @@ private EventTypeStatistics validateStatisticsUpdate(
         if (existing != null && newStatistics == null) {
             return existing;
         }
-        if (!Objects.equals(existing, newStatistics)) {
-            throw new InvalidEventTypeException("default statistics must not be changed");
+        if ((newStatistics.getReadParallelism() > nakadiSettings.getMaxTopicPartitionCount())
```

Comment:
This means that you can change it in a non meaningful way.

Previous: `read=2, write=4`. After: `read=3, write=4`. Do we want to prevent this?

Reply:
Since we take maximum of read and write parallilsm there is no change in partition count i.e., it will remain 4 however, the default statistics will still update read to 3.